python/tk_hiero_export/version_creator.py

```diff
@@ -128,6 +129,27 @@ def __init__(self, initDict):
         self._quicktime_path = None
         self._temp_quicktime = None
 
+    def addWriteNodeToScript(self, script, rootNode, framerate):
+        """
+        Override the default addWriteNodeToScript functionality so that we can add a SetNode before
+        the default writenodes get added to the script. The SetNode will allow us to later push our mov writenode
+        to this point so that it get parented correctly.
+        :param script:
```

Comment:
If Nuke's built-in method has decent param/return docs then include those, otherwise just go with your description above and skip the param and return docs entirely.

Reply:
yeah, sadly they don't I'll remove those, good point!